packages/server/src/channels/vonage/renderers/video.ts

```diff
@@ -20,7 +20,9 @@ export class VonageVideoRenderer implements ChannelRenderer<VonageContext> {
         type: 'video',
         text: undefined!,
         video: {
-          url: formatUrl(context.botUrl, payload.video)!
+          url: formatUrl(context.botUrl, payload.video)!,
+          // @ts-ignore
+          caption: payload.title!
```

Comment:
Why is there a ts ignore here?

Reply:
Because the typings that the Vonage SDK exposes are simply wrong or do not fit their own API!

Reply:
I'd prefer a cast to any over a ts-ignore in that case